### PR TITLE
feat: introduce granular permissions for workspaces

### PR DESCRIPTION
**Pull request by scavnickyj**
It is possible to set workspace permissions in a more granular/flat way now. The idea here is to map it similarly to how it is on BE.

Feature is behind enableGranularPermissions FF.

JIRA: F1-837
risk: low

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

### CODE REVIEW

**Review by kandl — DISMISSED** [review]
LGTM

File: libs/sdk-ui-ext/src/internal/components/dialogs/userManagementDialogs/RemoveIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export const RemoveIcon: React.FC<IRemoveIconProps> = ({ tooltipMessage, onClick, isDisabled = false }) => {
+    return (
+        <BubbleHoverTrigger showDelay={0} hideDelay={0} className="gd-grantee-item-delete">
+            <span
+                className={cx(
+                    "gd-grantee-item-icon gd-grantee-icon-trash gd-grantee-item-icon-right s-user-management-delete",
+                    { "is-disabled": isDisabled },
+                )}
+                onClick={isDisabled ? undefined : onClick}
+                aria-label="Share dialog grantee delete"
```
Comment (hackerstanislav) [inline libs/sdk-ui-ext/src/internal/components/dialogs/userManagementDialogs/RemoveIcon.tsx:24]: Localize?

**Review by hackerstanislav — APPROVED** [review]

File: libs/sdk-ui-ext/src/internal/components/dialogs/userManagementDialogs/RemoveIcon.tsx (hunk 1 above)
Comment (scavnickyj) [reply libs/sdk-ui-ext/src/internal/components/dialogs/userManagementDialogs/RemoveIcon.tsx:24]: no need for aria, will just make it more general

**Review by hackerstanislav — APPROVED** [review]

**Comment by scavnickyj** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2713/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Pull request merged**